org.mixedrealitytoolkit.spatialmanipulation/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "org.mixedrealitytoolkit.spatialmanipulation",
-  "version": "3.1.0-development",
```

Comment:
I don't think we need to increment the spatial manipulation version.  please check the others